src/launcher/index.ts

```diff
@@ -19,7 +19,7 @@ import { keyboard } from '../keyboard/keyboardController'
 import { cloneDeep, mergeWith } from 'lodash'
 import { FullScreenManager } from '../common/fullscreen'
 import { IFlowrDesktopConfig } from '../frontend/src/interfaces/IFlowrDesktopConfig'
-import { PlayerPosition } from '../frontend/src/interfaces/playerStore'
+import { WexondOptions } from "../wexond/main/app-window";
```

Comment:
We should add prettier to the commit hook as well
```suggestion
import { WexondOptions } from '../wexond/main/app-window'
```